C/plugins/south/dht11/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.4.0)
+
+project(dht11)
+
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -ggdb")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(WIRINGPILIBS -lwiringPi -lwiringPiDev)
+
+# Find source files
+file(GLOB SOURCES *.cpp)
+
+# Include header files
+include_directories(include ../../../services/common/include ../common/include ../../../common/include ../../../thirdparty/rapidjson/include /usr/local/include/)
+
+file(GLOB SOURCES "*.cpp" ../common/*.cpp)
+
+# Create shared library
+add_library(${PROJECT_NAME} SHARED ${SOURCES})
+target_link_libraries(${PROJECT_NAME} ${WIRINGPILIBS})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES SOVERSION 1)
```

Comment:
better to use major.minor version convention for SOVERSION

Reply:
SOVERSION 1 is being used in most other CMakeLists.txt files also in C/C++ part of the code.
@MarkRiddoch : Please provide your input here.